tools/parallel_requests.py

```diff
@@ -0,0 +1,217 @@
+# This script creates parallel processes to send predict requests
```

Comment:
Why is this something we need in the repo?

Reply:
This is to test drum server with nginx.
Sends requests, collects stats, etc.

Reply:
is it used?

Reply:
In test no.
But when I test, I can use this or other tools

Reply:
I just don't think people will use it after you productionize uwsgi but its fine to have for now and delete in a few months.